cmd/goal/ledger.go

```diff
@@ -52,3 +64,32 @@ var supplyCmd = &cobra.Command{
 		fmt.Printf("Round: %v\nTotal Money: %v microAlgos\nOnline Money: %v microAlgos\n", response.Round, response.TotalMoney, response.OnlineMoney)
 	},
 }
+
+var blockCmd = &cobra.Command{
+	Use:   "dumpblock <round>",
+	Short: "Show ledger block.",
+	Long:  "Show ledger block. In msgpack format or json format.",
```

Comment:
Remove periods, especially the one after 'block'.  `Dump` instead of `show` a is better term.
Perhaps:
"Dump block from the ledger"
"Dump a specific block from from the ledger, in msgpack or json format"